opencti-platform/opencti-front/tests_e2e/settings/parameters.spec.ts

```diff
@@ -15,7 +15,7 @@ test('Check EE activation', async ({ page }) => {
   await page.getByRole('button', { name: 'Enable' }).click();
   expect(page.getByText(/^Enterprise$/)).toBeVisible();
   await page.getByRole('button', { name: 'Disable Enterprise Edition' }).click();
-  expect(page.getByText(/^Enterprise$/)).toBeVisible();
+  expect(page.getByText(/^Community$/)).toBeVisible();
```

Comment:
unrelated to this PR but that triggers a legit question on making e2e tests with or without entreprise edition enabled

Reply:
and this seems to fail in drone actually